src/main/kotlin/calculator/Calculator.kt

```diff
@@ -33,4 +34,9 @@ class Calculator {
         }
         return splits
     }
+
+    companion object {
+        const val EXPRESSION_GROUP_SIZE = 2
+        const val EXPRESSION_START_PADDING = 1
```

Comment:
```suggestion
        private const val EXPRESSION_GROUP_SIZE = 2
        private const val EXPRESSION_START_PADDING = 1
```
클래스 내부에서만 접근한다면, private 접근자를 붙여보면 어떨까요?

Reply:
헉 ㅋㅋㅋ private는 다 붙혔다 생각했는데 남아 있었군요
https://github.com/next-step/kotlin-racingcar/pull/1142/commits/d5accceacdb069ef0b2c199502db33eb717c8588